### ISSUE TITLE
[Bug] Magic Link signup not working

### ISSUE THREAD

**Issue by TheCarpetMerchant** [AppFlowy-IO/AppFlowy#6129]
### Bug Description

First of, in Gmail when clicking the link it opens appended to "https://www.google.com/url?q=".

But the main problem is that when actually accessing the given url and clicking on the popup to open with "appflowy-flutter", this is what happens :
![Screenshot (310)](https://github.com/user-attachments/assets/a3d7c71c-ad03-489f-a662-4f166652f1c7)
 
The bottom bar pops up, stays there a few seconds, and nothing else happens.

### How to Reproduce

- Download the installer
- Install under the default folder
- Open and set a gmail address
- Click continue
- In the email, right-click the log in button and copy link
- Open the link in a new tab
- Click "Open" when asked to open the link with appflowy-flutter
- Bug occurs.

### Expected Behavior

Log in / Sign up with magic link works

### Operating System

Windows 10.0.19045 Build 19045

### AppFlowy Version(s)

0.6.7.2

**Comment by wkoehn** [comment]
I can't even get the android or windows apps to send a magic link. The email never shows up. Also tried "Continue with Google" and this times out with this error message:

![image](https://github.com/user-attachments/assets/31b2ced8-dbad-409c-8acc-f2f99fcf028b)

**Comment by Nocturnal-ru** [comment]
same problem

**Comment by BA-24** [comment]
Just got introduced to this program 30 minutes ago but previous versions (0.6.7.1, 0.6.6) don't seem to work either on my Windows machine. Looks like back-end issues?

**Comment by OctoShotG** [comment]
Same issue, also not getting any links emailed
RPM Linux app on Fedora 40, latest Flatpak

**Comment by khorshuheng** [comment]
The backend service was unavailable, which has been resolved. Does the issue still persist?

**Comment by OctoShotG** [comment]
Was able to login successfully

**Comment by Xazin** [comment]
How about @wkoehn @TheCarpetMerchant @Nocturnal-ru and @BA-24 , do any of you still experience this issue?

**Comment by max-kazak** [comment]
I have the same issue on my system as well (flatpak version on Fedora 40).
Android app works using the same method.

**Comment by annieappflowy** [comment]
@max-kazak , did you use Firefox?

**Comment by max-kazak** [comment]
@annieappflowy, yes I usually use firefox. However, I've tried opening the magicling in chromium as well with the same results.

**Comment by candideu** [comment]
Same issue for me...

I had to create the account on https://beta.appflowy.cloud first, then generate the Magic Links using the Android app, and then click on the magic link on my desktop.

The desktop app was unable to generate the links.

**Comment by 0x00000oak** [comment]
Seriously. Just let us sign in with password.

**Comment by ToS0** [comment]
I get the server-side error, that my SMTP credentials do not work, but they do work with other apps:
`    "component":"api",
    "error":"Error sending magic link email: 535 5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials ffacd0b85a97d-38621909754sm10076973f8f.80 - gsmtp",
    "level":"error",
    "method":"POST",
    "msg":"500: Error sending magic link",
    "path":"/magiclink",
    "referer":"appflowy-flutter://",
    "remote_addr":"172.18.0.7",
`

**Comment by beep-beep-beep-boop** [comment]
I think adding the option to at least paste the link it sends you and extracting the token from that might be decent. Or just also writing the token in the email and letting you paste it in, like a lot of things do. Depending on the link handling always working will mean there's a lot more cases where signing-in might be broken.

**Comment by khorshuheng** [comment]
@beep-beep-beep-boop Yes, we are adding the support for OTP soon.

**Comment by annieappflowy** [comment]
We have supported two alternative ways to sign in:
1. login with OTP 
2. paste the magic link directly in your browser (thanks, @beep-beep-beep-boop )

We will soon support login with password (thanks, @oaklsb )
